aws-lambda-layer/Dockerfile

```diff
@@ -1,14 +1,19 @@
-FROM lambci/lambda:build-ruby2.5
+FROM amazonlinux:latest
 
 ARG FREETDS_VERSION=1.1.6
 
 ENV INSTALLDIR='/tmp/freetds'
+ENV TASKDIR='/var/task'
```

Comment:
I don't think this needs to be parameterized with an env var... we don't yet need to override it, so it can probably be hardcoded